core/app/c/(public)/[communitySlug]/public/forms/[formSlug]/fill/ExternalFormWrapper.tsx

```diff
@@ -1,42 +1,16 @@
 "use client";
```

Comment:
This isn't a useful diff— `ExternalFormWrapper` has totally changed. See diagram for info.

This and `PubEditorWrapper` both serve as client-side wrappers around `PubEditorClient`. They need a wrapper so that we can pass a function `onSuccess` as a prop, since both paths want to handle success states differently.